src/views/UserOverView.vue

```diff
@@ -169,9 +197,9 @@ const [canGoIntoDebt, canGoIntoDebtAttrs] = defineField('canGoIntoDebt', {});
 const isActiveFilter: Ref<boolean> = ref(true);
 const ofAgeFilter: Ref<boolean> = ref(true);
 const visible: Ref<boolean> = ref(false);
-const loading = ref(false);
```

Comment:
I would change this to isLoading for consistency

Reply:
Sure, fixed now.